src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlCommandSimplifier.cs

```diff
@@ -0,0 +1,97 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.SqlServer;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Search
+{
+    internal static class SqlCommandSimplifier
```

Comment:
Since we generate the sql all with our own code, I'm not sure I understand why we also string manipulate the string output?
Now we have Expressions => Writers => Sql Generator => Regex